8.5-jre8/server.xml

```diff
@@ -71,7 +71,11 @@
     <Connector port="${port.http}" protocol="HTTP/1.1"
                connectionTimeout="20000"
                redirectPort="8443"
-               maxHttpHeaderSize="16384" />
+               maxHttpHeaderSize="16384"
+             compression="on"
+             compressionMinSize="1024"
+             noCompressionUserAgents="gozilla, traviata"
+             compressableMimeType="text/html,text/xml" />
```

Comment:
Good change.. Would you mind submitting a PR for the dev branch (instead of master)? We recently introduced 9.0-jre8 alongside 8.5-jre8 (in the dev branch). You might want to make a similar change to 9.0.

I know this PR has been open for a while, I'll be happy to make these changes myself.